### ISSUE TITLE
Not working properly

### ISSUE THREAD

**Issue by cbasavaraj** [christopher5106/FastAnnotationTool#6]
Hi Chris,

Thanks for this tool, but doesn't seem to work properly for me. I'm on macOS Sierra 10.12.4. When I click the first time, it draws a box already centred on the click. Every subsequent click, it draws a back whose diagonal is from the center of the previous box to the point of this click. The arrow keys, with our without FN, also don't work as they are supposed to. Right arrow just goes to the next image, for example.

Probably again an issue with the key codes? Thanks for any help.

**Comment by christopher5106** [comment]
No, it is the diagonal mode to draw faster the rectangle from the diagonal. You can switch this mode off.
